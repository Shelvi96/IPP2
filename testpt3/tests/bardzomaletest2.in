NEW ID

:497>2344769::
680:47>99743
:4;8093>3;
;6;9>98298
24135878>5041559298
8269>7515
6>:723493978
81>9385
8>8:;33
89;;;61>23:3:69
;882814>1
:43614743>291503
299089829>::3
806>522999;
28:69>7:180929;
;7;07964>011907496:
6987589>:89
3012>065
;624661>8
@5;:;87;:70;3459654097:;
@4548177;7527415;5869148:0887:7
@35
@;:;696335;82
@1:9341;25;:19:0033412:
@066593708:2;27:819;2796;0935
@6044;
@161
@2;425;42421:1240668;:246638
@9:1399571:1546
@580926558;9354;551:9;86
@18;6452917;
@54024:9160:429155;3606
@4;;64
@5;31;991:;8;901:5
@78518;:1:17
@;056584
