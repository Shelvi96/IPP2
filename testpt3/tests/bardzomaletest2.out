11311000000
1019663113981482080
0
0
435257469
8844407150077543
0
0
212000000000000
8
54897322612
0
1120012000
0
8840
0
0
